DigitalOffice/Services/UserService/Startup.cs

```diff
@@ -43,10 +53,41 @@ public void ConfigureServices(IServiceCollection services)
             ConfigureMappers(services);
         }
 
+        private void ConfigRabbitMq(IServiceCollection services)
+        {
+            string appSettingSection = "ServiceInfo";
+            string serviceId = Configuration.GetSection(appSettingSection)["ID"];
+            string serviceName = Configuration.GetSection(appSettingSection)["Name"];
+
+            services.AddMassTransit(x =>
+            {
+                x.AddConsumer<UserConsumer>();
+
+                x.UsingRabbitMq((context, cfg) =>
+                {
+                    cfg.Host("localhost", host =>
+                    {
+                        host.Username(serviceName);
```

Comment:
I think, that username should contain serviceId, because there can be several services with the same name.